src/styles/globals.scss

```diff
@@ -299,3 +299,27 @@
     margin-bottom: 0 !important;
   }
 }
+
+.IntroCards {
+  display: grid;
+  gap: pxToRem(12px);
+  grid-template-columns: 1fr 1fr;
+
+  .Card {
+    display: flex;
+    flex-direction: column;
+    gap: pxToRem(8px);
+    justify-content: space-between;
+
+    :nth-child(2) {
+      flex: 1;
+    }
+  }
+}
```

Comment:
This was moved to its own component (`InfoCards`). Let's make it's not in global styles.